src/schema/objects/extensions.yaml

```diff
@@ -109,6 +118,11 @@ fif:
   display_name: Functional Imaging File Format
   description: |
     An MEG file format used by Neuromag, Elekta, and MEGIN.
+funcgii:
+  value: .func.gii
+  display_name: Functional GIFTI
+  description: |
+    GIFTI surface data format file.
```

Comment:
Given that it's possible we should be using `.time.gii` instead, I think we should not hold this PR up on adding `.func.gii`.

```suggestion
```